.github/workflows/CI.yaml

```
@@ -56,7 +45,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: [ '8.10.7', '9.2.2' ]
+        ghc: [ '8.10.7', '9.2.3' ]
```

I think 9.2.4 is out now too if you're interested in using that instead.
```suggestion
        ghc: [ '8.10.7', '9.2.4' ]
```